data/structs.go

```diff
@@ -0,0 +1,51 @@
+package data
+
+import (
+	"bufio"
+	"encoding/json"
+	"errors"
+	"os"
+)
+
+var (
+	ErrUnknownType = errors.New("unknown data type")
+)
+
+type DataType uint8
+
+const (
+	BlockBidAndTraceData DataType = iota
+)
+
+type exportRequest struct {
+	dt     DataType
+	data   any
+	caller string
+	err    chan error
+}
+
+type exportFiles struct {
+	BlockBidAndTrace *os.File
```

Comment:
Nothing in this file depends on the type, so why bother?

Reply:
I don't understand the suggestion

Reply:
Make this universal don't call it "BlockBidAndTrace" but rather "Handler".  ro something